x-pack/legacy/plugins/lens/public/indexpattern_plugin/operation_definitions/date_histogram.tsx

```diff
@@ -79,6 +79,27 @@ export const dateHistogramOperation: OperationDefinition<DateHistogramIndexPatte
       },
     };
   },
+  isTransferable: (column, newIndexPattern) => {
+    const newField = newIndexPattern.fields.find(field => field.name === column.sourceField);
+
+    if (!newField || !newField.aggregatable) {
+      return false;
+    }
+
+    if (!newField.aggregationRestrictions) {
+      return true;
+    }
+
+    if (newField.aggregationRestrictions.date_histogram) {
+      const restrictions = newField.aggregationRestrictions.date_histogram;
+      return (
+        restrictions.calendar_interval === column.params.interval &&
+        restrictions.time_zone === column.params.timeZone
+      );
+    } else {
```

Comment:
For all of the isTransferable functions, I think it's important to match field names and types across index patterns, not just names.

For date histogram specifically, you have not considered the `fixed_interval` case- I think that needs to be included